compute/rn/release-information/release-notes-22-06-update1.adoc

```diff
@@ -0,0 +1,55 @@
+== 22.06 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Codename
+|Kepler, 22.06 Update 1
+//Tentative date
+|Release date
+|Jul 27, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
```

Comment:
@Pubs-MV where do you think is the right place to mention new supported orchestrations like TKGI and [GKE arm](https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39581)?

cc @grevach

Reply:
we added a new bullet to the top so it is the highlight for the update @MorShabiPAN